cg/store/crud/read.py

```diff
@@ -612,8 +612,8 @@ def get_ready_made_library_expected_reads(self, case_id: str) -> int:
 
     def get_samples_by_customers_and_pattern(
         self, *, customers: list[Customer] | None = None, pattern: str = None
-    ) -> list[Sample]:
-        """Get samples by customer and sample internal id  or sample name pattern."""
+    ) -> tuple[list[Sample], int]:
+        """Get samples by customer and sample internal id or sample name pattern."""
```

Comment:
```suggestion
        """Get samples by customer and sample internal id or sample name pattern.
        This query is used in the front end next to the samples it also return the total number of samples used to determine the pagination.
        """
```

Reply:
Is this usually part of the documentation? To me, it does not make sense that the backend should know what the FE will use it for. I would probably write `Also returns the total number of samples to support pagination`.

Reply:
I added the docstring but I think needs some polishing, will look at it later